### DISCUSSION TITLE
Splatoon3カスタムの役職振り分け

### DISCUSSION THREAD

**Discussion by Tooooomo** [KU-eSports/kuest-streaming#50] [General]
役職振り分け #41
 
***

運営メンバーの中で役職を振り分けます。
イカに必要な役職を列記していくので、担当できる部分・担当したい部分を教えてください。
複数人での並行作業を想定しているので重複可です。

## 前日まで
- 参加者集め @N4K4eight ほか
- 配信デザインラフ @Tooooomo 
- 配信デザイン実装(動画) @Tooooomo 
- 配信デザイン実装(css) @NYA0321 @TAKA00000000 
- DiscordBot開発 @secchanu 

## 当日
- カメラ(キャプチャーボード必要) @secchanu 
- 実況 @N4K4eight 
- 解説 @Tooooomo 
- 配信 @ykmtsd 
- VC誘導 @NYA0321

**Comment by secchanu** [comment]
カメラと配信できます
昔は両方やってたけど腕の本数的に厳しいです

**Comment by Tooooomo** [comment]
実況・解説できます
人数の集まり~~とせっちゃんの負担~~次第では配信に回ります

**Comment by Mickey0321** [comment]
当日はvc誘導ができます。(ほかもできるようにすることはできます)
前日までは、ルール決定の項ができます。
その他ソース書くとこはせっちゃさんの補助輪付きだったらいけると思います。
僕よりせっちゃさんの方が僕の技量わかってると思うので必要なら

**Comment by N4K4eight** [comment]
カメラor実況やりたいです
キャプボはいずれ買う予定なので、配信もやろうと思えば可能です

**Comment by Ere4CKywyEHz** [comment]
配信やりたいです

**Comment by Tooooomo** [comment]
当日の役職とりあえず割り振っておきました
にゃあさんはコードとかcssのほうに回ってもらうことになりそうなので前日までの役職は保留で
valoカスタムのコードとか見てみてどこまでできそうか把握できるなら教えてくれるとめっちゃ助かる

**Reply by Mickey0321** [reply]
どこまでできるのか...
正直わからないです

**Reply by Tooooomo** [reply]
了解
たぶん実際に手つけてみないとお互い分からないだろうし共同作業になるかもっていう心構えはしといてもらえたら
cssは触ったことあるんだっけ？

**Reply by Mickey0321** [reply]
cssは多少は

**Reply by Tooooomo** [reply]
それならトーク画面とか待機画面とかの実装を手伝ってもらうことになると思う
前回でいうとkuest-streaming/src/browser以下のdashboardとgraphicsの中身にあたるかな
せっちゃんとの相談次第で担当箇所変わる可能性があるから確実ではないけど目通してみることをおススメします

**Reply by secchanu** [reply]
今回試合結果とか取れなさそうだからextensionはほとんど触らないだろうしどっちにしろbrowser以下見といてもらえると

**Comment by secchanu** [comment]
これルール先に決めないとBotが必要かどうかとかいろいろ関わってくると思うんだけど
優先度高くない？
